src/Services/src/ErrorDialog/IErrorDialog.cs

```diff
@@ -7,5 +7,6 @@ namespace Tanzu.Toolkit.Services.ErrorDialog
     public interface IErrorDialog
     {
         void DisplayErrorDialog(string errorTitle, string errorMsg);
+        void DisplayWarningDialog(string warningTitle, string warningMsg);
```

Comment:
My $.02 is that the param names like `warningTitle` are redundant.  Just `title` is sufficient since the function/method name gives the "warning" context.

Reply:
done!